core-feature-pack/src/main/resources/content/bin/standalone.sh

```diff
@@ -82,7 +82,7 @@ if $cygwin ; then
 fi
 
 # Setup JBOSS_HOME
-RESOLVED_JBOSS_HOME=`cd "$DIRNAME/.."; pwd`
+RESOLVED_JBOSS_HOME=`cd "$DIRNAME/.." >/dev/null; pwd`
```

Comment:
if CDPATH is set, every cd command echoes current directory, so you know that CDPATH is effect.
Which can cause that RESOLVED_JBOSS_HOME can include wrong value.